git/remote.go

```diff
@@ -100,3 +100,14 @@ func AddRemote(name, u string) (*Remote, error) {
 		PushURL:  urlParsed,
 	}, nil
 }
+
+//CheckRemoteExist it's will verify with there is a remote configured at directory
+func CheckRemoteExist() (bool, error) {
+	out, err := exec.Command("git", "remote", "-v").Output()
+
+	if err != nil || out != nil {
+		return false, nil
+	}
+
+	return true, nil
```

Comment:
This will only ever return `nil` as an error. It's fine for this to only return a bool and return `false` if the git command fails (as it's doing now).